README.md

```diff
@@ -43,25 +35,68 @@ on and off on the fly during cluster operation. This allows you to put
 different types of workloads (e.g., compute- and transaction-intensive ones) on
 different physical servers with dedicated hardware.
 
-## Installation
+Tarantool Cartridge has an external utility called `cartridge-cli` which provides you
+with utilities and templates to help:
 
-```shell
-you@yourmachine $ tarantoolctl rocks install cartridge
+* easily set up a development environment for your applications;
+* plug the necessary Lua modules;
+* pack the applications in an environment-independent way: together with
+   module binaries and Tarantool executables.
+
+## Getting Started
+
+### Create first application
+
+To get a template application that usage Tarantool Cartridge and run it we need
+to install `cartridge-cli` utility. We are suggested that [`tarantool`](https://www.tarantool.io/en/download/) are already installed.
+
+```sh
+tarantoolctl rocks install cartridge-cli
 ```
 
-This will install ``cartridge`` to ``~/.rocks``.
+Now, you can create your first Tarantool Cartridge application:
+
+```sh
+.rocks/bin/cartridge create --name myapp
+```
+
+Application was create in `./myapp`, welcome:
+
+```sh
+cd myapp
+```
+
+Let's install application dependencies locally:
+
+```sh
+.rocks/bin/cartridge build
+```
 
-## Usage
+Start our application:
 
-This repository contains the framework itself.
+```sh
+.rocks/bin/cartridge start
+```
+
+That's all! You can visit localhost:8081 and see your application Admin Web UI:
 
-To build a Tarantool cluster app, you'll also need the
-[cartridge-cli](https://github.com/tarantool/cartridge-cli/) utility.
+![Tarantool Cartridge Web UI](./images/cartridge-ui.png)
 
-See a step-by-step
+
+### Next stages
+
+**See** a step-by-step
 [getting started guide](https://github.com/tarantool/cartridge-cli/blob/master/examples/getting-started-app/README.md)
 in the ``cartridge-cli`` repository.
 
+## Installation (for advanced users)
```

Comment:
It's just a rock module installation, nothing advanced, is it?

Reply:
I'm try to separate our nobbie user from our advanced who understand what rock is it and how to use it further.

Reply:
It just confuses.

User should understand what rock module is, at least to add some new modules to rockspec on application development.